content/run.md

```diff
@@ -240,3 +240,111 @@ Now you can perform the final steps to build and submit the case to the job queu
 
     [~/cases/SCA_SpRun]$ ./case.build
     [~/cases/SCA_SpRun]$ ./case.submit
+
+### Example regional run bash scripts
+
+`````{tabs}
+  ````{tab} 0.25&deg; grid
+
+  The following example requires read access to
+  `/cluster/projects/nn9373k/data_regional_SBS` on Fram.
+  ```bash
+      # Project accounting
+      export CESM_ACCOUNT=nn9373k
+      export PROJECT=nn9373k
+
+      # Paths: CLM, CIME and NetCDF
+      export CTSMROOT=$HOME/ctsm_fates_emerald
+      export CIMEROOT=$CTSMROOT/cime
+      export INC_NETCDF=/cluster/software/VERSIONS/netcdf.intel-4.3.3.1/
+      export LIB_NETCDF=/cluster/software/VERSIONS/netcdf.intel-4.3.3.1/
+      export NETCDF_ROOT=/cluster/software/VERSIONS/netcdf.intel-4.3.3.1/
+
+      # Input
+      export CESM_DATA=/cluster/shared/noresm/inputdata
+      export MYCTSMDATA=$USERWORK/inputdata
+      export REG_DATA=/cluster/projects/nn9373k/data_regional_SBS
```

Comment:
Use the same environment variables explained [here](https://metos-uio.github.io/CTSM-Norway-Documentation/run/#run-a-regional-case) and also at later stages in the script.